common/src/main/java/io/seata/common/DefaultValues.java

```diff
@@ -60,6 +60,8 @@ public interface DefaultValues {
     boolean DEFAULT_ENABLE_RM_CLIENT_BATCH_SEND_REQUEST = true;
     boolean DEFAULT_ENABLE_TC_SERVER_BATCH_SEND_RESPONSE = false;
 
+    boolean DEFAULT_CLIENT_CHANNEL_CHECK_FAIL_FAST = false;
```

Comment:
`FAIL_FAST` defaultValue is `true` or better ? 

Only saga mode is possible without depend serverList